mergify_engine/web/api/queues.py

```diff
@@ -315,6 +323,38 @@ async def repository_queues(
     return queues
 
 
+@router.get(
+    "/repos/{owner}/{repository}/queues/configurations",  # noqa: FS003
+    summary="Get merge queues configurations",
```

Comment:
```suggestion
    summary="Get merge queues configuration",
```

The configuration of the queues :)